### PR TITLE
Add note about restrictions svc of LB type with Kuryr

### PR DESCRIPTION
**Pull request by MaysaMacedo**
When Kuryr is used creation of Services with Load Balancer type
are restricted to when a Router and External Connectivity exists
on the Cluster.

### CODE REVIEW

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 1]
```diff
@@ -11,7 +11,10 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support NodePort services.
+{product-title} with Kuryr SDN does not support NodePort services. Also, if
+the Machines Subnet is not connected to a Router or it's, but the Router is
```
Comment (maxwelldb) [inline modules/installation-osp-kuryr-known-limitations.adoc:15]: What is "or it's" here?

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 2]
```diff
@@ -11,7 +11,10 @@ Using {product-title} with Kuryr SDN has several known limitations.
...
+not connected to the external Network, Kuryr won't wire Services of
```
Comment (maxwelldb) [inline modules/installation-osp-kuryr-known-limitations.adoc:16]: "wire" -> "connect" ?

**Review by maxwelldb — CHANGES_REQUESTED** [review]

File: modules/installation-osp-kuryr-known-limitations.adoc (hunk 1 above)
Comment (MaysaMacedo) [reply modules/installation-osp-kuryr-known-limitations.adoc:15]: I meant "or it's connected". Let me rephrase it.

File: modules/installation-osp-kuryr-known-limitations.adoc (hunk 2 above)
Comment (MaysaMacedo) [reply modules/installation-osp-kuryr-known-limitations.adoc:16]: perhaps "handle" can fit better.

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 3]
```diff
@@ -11,7 +11,10 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support NodePort services.
+{product-title} with Kuryr SDN does not support NodePort services. Also, if
```
Comment (maxwelldb) [inline modules/installation-osp-kuryr-known-limitations.adoc:14]: > Also, if the Machines Subnet is not connected to a Router or it’s connected, but the Router has no external gateway set, Kuryr won’t create Floating IPs for Services with LoadBalancer type.

-> 

> [line break]
> If the machines subnet is not connected to a router, or if the subnet is connected, but the router has no external gateway set, Kuryr cannot create floating IPs for load balancer services.

**Review by maxwelldb — CHANGES_REQUESTED** [review]
Thoughts on this edit?

**Comment by maxwelldb** [comment]
Which versions should this apply to?

File: modules/installation-osp-kuryr-known-limitations.adoc (hunk 3 above)
Comment (MaysaMacedo) [reply modules/installation-osp-kuryr-known-limitations.adoc:14]: Sounds good.

**Comment by MaysaMacedo** [comment]
> Which versions should this apply to?

4.6

File: modules/installation-osp-kuryr-known-limitations.adoc (hunk 3 above)
Comment (MaysaMacedo) [reply modules/installation-osp-kuryr-known-limitations.adoc:14]: I modified  "load balancer services" to "LoadBalancer services" to give a better indication that it's the type of the Service.

Comment (maxwelldb) [reply modules/installation-osp-kuryr-known-limitations.adoc:14]: >[line break]
>If the machines subnet is not connected to a router, or if the subnet is connected, but the router has no external gateway set, Kuryr cannot create floating IPs for LoadBalancer services.

That, then?

Comment (maxwelldb) [reply modules/installation-osp-kuryr-known-limitations.adoc:14]: Committing directly.

**Comment by maxwelldb** [comment]
@MaysaMacedo https://github.com/openshift/openshift-docs/pull/25508/files#diff-b6143035dd5fcf10335d556668c5a745

**Comment by MaysaMacedo** [comment]
Looks good @maxwelldb .

**Comment by maxwelldb** [comment]
@MaysaMacedo TY! Do you have anyone from QE who's familiar with this and could +1 it?

**Comment by eurijon** [comment]
It looks good from QE side!

**Comment by MaysaMacedo** [comment]
> It looks good from QE side!

Thanks, Jon.

**Review by maxwelldb — APPROVED** [review]

**Comment by ahardin-rh** [comment]
LGTM! You just need to squash the two commits down into one. Thanks!

**Comment by maxwelldb** [comment]
Thanks!
